admin_guide/out_of_resource_handling.adoc

```diff
@@ -47,10 +48,13 @@ description column based on the node summary API.
 |`memory.available` = `node.status.capacity[memory]` - `node.stats.memory.workingSet`
 |===
 
+[NOTE]
+====
 In future releases, the node will support the ability to trigger eviction
 decisions based on disk pressure. Until then, use
 xref:../admin_guide/garbage_collection.adoc#admin-guide-garbage-collection[container
 and image garbage collection].
+====
 
 [[out-of-resource-eviction-thresholds]]
 === Eviction Thresholds
```

Comment:
With the below, and this is said a few times, but we're saying "The node supports the ability to X" a lot. I take that to mean something needs to be enabled in order to use it. Is that the case? If it's by default, it should just be saying "The node can X".